app/controllers/comments_controller.rb

```diff
@@ -0,0 +1,25 @@
+class CommentsController < ApplicationController
+  def new
+    @post = Post.find(params[:post_id])
+    @user = @post.author
+    @comment = Comment.new
+    render :new, locals: { comment: @comment, post: @post }
+  end
+
+  def create
+    @user = @current_user
+    @post = Post.find_by(id: params[:post_id])
+
+    puts "params[:post_id]: #{params[:post_id]}"
+    puts "@post: #{@post.inspect}"
+
+    if @post.nil?
+      redirect_to root_path
+    else
+      @comment = Comment.new(user_id: @user.id, post_id: @post.id, comment: params[:comment])
+
```

Comment:
![image](https://github.com/sumon766/techbright/assets/93039359/dca5a1b6-5dbe-44ac-a7e3-c3ab716b562d)

- [ ] It would be nice if you fix this issue because I have faced it while adding a **comment** on a post 😨 Kindly fix it to make your app work as expected 👍